### PR TITLE
Noticeable components

### PR DESCRIPTION
**Pull request by feyzullah**
### Actions
* Update Email Subscription
* Create Publication
* Create Email Subscription
* Delete Email Subscription
* Search Email Subscription

Triggers
* Email Subscription Created
* Email Subscription Deleted
* Publication Created
* Publication Updated
* Publication Deleted

### CODE REVIEW

File: components/noticeable/actions/create-email-subscription/create-email-subscription.mjs [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import app from "../../noticeable.app.mjs";
+
+export default {
+  type: "action",
+  key: "noticeable-create-email-subscription",
+  version: "0.0.2",
```
Comment (michelle0927) [inline components/noticeable/actions/create-email-subscription/create-email-subscription.mjs:6]:
```suggestion
  version: "0.0.1",
```

File: components/noticeable/actions/create-publication/create-publication.mjs [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import app from "../../noticeable.app.mjs";
+
+export default {
+  type: "action",
+  key: "noticeable-create-publication",
+  version: "0.0.26",
```
Comment (michelle0927) [inline components/noticeable/actions/create-publication/create-publication.mjs:6]:
```suggestion
  version: "0.0.1",
```

File: components/noticeable/actions/delete-email-subscription/delete-email-subscription.mjs [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import app from "../../noticeable.app.mjs";
+
+export default {
+  type: "action",
+  key: "noticeable-delete-email-subscription",
+  version: "0.0.4",
```
Comment (michelle0927) [inline components/noticeable/actions/delete-email-subscription/delete-email-subscription.mjs:6]:
```suggestion
  version: "0.0.1",
```

File: components/noticeable/actions/search-email-subscription/search-email-subscription.mjs [hunk 4]
```diff
@@ -0,0 +1,65 @@
+import app from "../../noticeable.app.mjs";
+import { getEmailSubscriptionsQuery } from "../../common/queries.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  type: "action",
+  key: "noticeable-search-email-subscription",
+  version: "0.0.5",
```
Comment (michelle0927) [inline components/noticeable/actions/search-email-subscription/search-email-subscription.mjs:8]:
```suggestion
  version: "0.0.1",
```

File: components/noticeable/actions/update-email-subscription/update-email-subscription.mjs [hunk 5]
```diff
@@ -0,0 +1,66 @@
+import app from "../../noticeable.app.mjs";
+
+export default {
+  type: "action",
+  key: "noticeable-update-email-subscription",
+  version: "0.0.4",
```
Comment (michelle0927) [inline components/noticeable/actions/update-email-subscription/update-email-subscription.mjs:6]:
```suggestion
  version: "0.0.1",
```

File: components/noticeable/sources/publication-updated/publication-updated.mjs [hunk 6]
```diff
@@ -0,0 +1,38 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  key: "noticeable-publication-updated",
+  name: "New Publication Updated Event",
+  description: "Emit new events when a new publication created. [See the docs](https://graphdoc.noticeable.io/publication.doc.html)",
```
Comment (michelle0927) [inline components/noticeable/sources/publication-updated/publication-updated.mjs:7]:
```suggestion
  description: "Emit new events when a new publication is updated. [See the docs](https://graphdoc.noticeable.io/publication.doc.html)",
```

**Comment by feyzullah** [comment]
Hi @michelle0927, sorry for the mess. Can you please check once more?

**Review by michelle0927 — APPROVED** [review]
Approved!

**Comment by vunguyenhung** [comment]
Hello everyone, I have tested this PR and there're some test cases failed or needed improvement. 

Please check the test report below for more information
[Noticeable_5323_2138.pdf](https://github.com/PipedreamHQ/pipedream/files/10536775/Noticeable_5323_2138.pdf)

**Comment by feyzullah** [comment]
Hi @vunguyenhung, about `Create Publication` action, the problem must have been solved. Can you please check once more?
About `Email Subscription Updated` trigger, I noticed that Noticeable services does not update `updatedAt` field and we are missing the updates. To make it work, we need to store whole object and compare all fields. And it may be costly. My suggestion is to remove this source completely until Noticeable team solves the problem. What would be your suggestion?

**Comment by vunguyenhung** [comment]
Hello @feyzullah, I've tested again and the Create Publication passed! I'm good to skip the source `Email subscription Updated`, so I'll move this to ready for release! Kindly remove the source then get approval and merge it. Thank you!

Test report
[Noticeable_5323_2012.pdf](https://github.com/PipedreamHQ/pipedream/files/10569277/Noticeable_5323_2012.pdf)

**Review by andrewjschuang — APPROVED** [review]

**Pull request merged**